.sphinx/_templates/footer.html

```diff
@@ -101,5 +101,21 @@
 
 
     </div>
+    <div>
+      {% if github_url and github_folder and pagename and page_source_suffix %}
```

Comment:
We should add one more variable here, to make it possible to turn this feature off. Some teams might not need it, or they might have a private repository and don't want to display the contributors.

So we should have a variable in `custom_conf.py`, for example, `display_contributors`. This variable should be set to True by default, but it should be possible to set it to False instead.